ts/components/screens/help/HelpButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
```

Comment:
It is used only by `BaseHeader`. Is it useful have it in a dedicated file?

Reply:
I think it improves code readability.

Reply:
Well, we do lose the comfort of being able to open a component file with `cmd + P` (on VSCode/macOS)

Still, is this is the convention for the codebase, we should stick to it 👍